.vscode/settings.json

```diff
@@ -1 +1,3 @@
-{}
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
+}
```

Comment:
This makes sure vscode uses the repo version of ts instead of a different one in vscode. This is important because some of the type inference only works with the newer versions of typescript, so the dev experience would be much worse if there is accidentally an old version configured

Reply:
we should add this to the templates too?